spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/MemoryData.scala

```diff
@@ -278,4 +278,18 @@ private[mkldnn] object MemoryData {
     memory.setPrimitiveDescription(memoryPrimDesc)
     memory
   }
+
+  def operationWant2(primDesc: Long, queryType: Int, index: Int): NativeData = {
```

Comment:
why operationWant2? need a better name

Reply:
Yes, the method has been renamed operationWantWithIndex.